docs/sources/administration/provisioning/_index.md

```diff
@@ -364,8 +364,565 @@ providers:
 
 > **Note:** To provision dashboards to the General folder, store them in the root of your `path`.
 
+## Alerting
+
+You can manage alert objects in Grafana by adding one or more YAML or JSON
+configuration files in the [`provisioning/alerting`]({{< relref "../../setup-grafana/configure-grafana/" >}})
+directory. Those files will be applied when starting Grafana. When Grafana
+is running, it's possible to do a hot reload using the
+[Admin API]({{< relref "../../developers/http_api/admin/#reload-provisioning-configurations" >}}).
+
+### Rules
+
+Creation
+
+```yaml
+# config file version
+apiVersion: 1
+
+# List of rule groups to import or update
+groups:
+  # <int> organization id, default = 1
+  - orgId: 1
+    # <string, required> name of the rule group
+    name: my_rule_group
+    # <string, required> name of the folder the rule group will be stored in
+    folder: my_first_folder
+    # <duration, required> interval that the rule group should evaluated at
+    interval: 60s
+    # <list, required> list of rules that are part of the rule group
+    rules:
+      # <string, required> unique identifier for the rule
+      - uid: my_id_1
+        # <string, required> title of the rule that will be displayed in the UI
+        title: my_first_rule
+        # <string, required> which query should be used for the condition
+        condition: A
+        # <list, required> list of query objects that should be executed on each
+        #                  evaluation - should be obtained trough the API
+        data:
+          - ...
```

Comment:
IMHO we should really provide an example here, as long as it is. This is by far the hardest to understand portion of rules.

Reply:
Yes, will add it.